lib/MusicBrainz/Server/Entity/EditNote.pm

```diff
@@ -79,6 +82,18 @@ sub localize {
     return $text;
 }
 
+around TO_JSON => sub {
+    my ($orig, $self) = @_;
+
+    my $json = $self->$orig;
+    $json->{editor_id} = $self->editor_id + 0;
+    $json->{editor} = $self->editor->TO_JSON;
+    $json->{post_time} = datetime_to_iso8601($self->post_time);
+    $json->{text} = $self->editor_id == 4 ? $self->localize : ($self->text | format_editnote);
```

Comment:
`|` is TT syntax -- that does something different in Perl (bitwise OR).

We might want to name this `formatted_text` too to distinguish it from the raw text attribute
```suggestion
    $json->{formatted_text} = $self->editor_id == 4 ? $self->localize : format_editnote($self->text);
```